src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -197,69 +198,78 @@ const AdminDashboard = (props) => {
     `,
   }
 
-// console.log(UNPAID_SESSIONS)
+  const arr = [];
+
+  const dummy = {
+    course: {startTime: "09:00:00", endTime: "11:00:00", hourlyTuition: 100,},
+    lastPaidSessionDatetime: "2020-06-12T22:15:00+00:00",
+    paymentList: [
+      {
+        parent: {
+          user: {
+            firstName: "Stop", lastName:"Nesting", id: "4",
+            __typename: "ParentType"
+          }
+        }
+      }
+    ],
+  }
+
+  const janetmann = {
+    course: {startTime: "15:15:00", endTime: "16:15:00", hourlyTuition: 100,},
+    lastPaidSessionDatetime: "2020-06-14T22:15:00+00:00",
+    paymentList: [
+      {
+        parent: {
+          user: {
+            firstName: "Janet", lastName:"Mann", id: "3",
+          }
+        }
+      }
+    ],
+  }
 
- const unpaidSessionsData = useQuery(QUERIES["unpaidsessions"])
-//  console.log(unpaidSessionsData)
+
+
+
+ const {data, loading, error} = useQuery(QUERIES["unpaidsessions"])
+//  console.log(data)
+ 
  
 const getTime = (time) => {
-  const strTime = String(time);
-  const minutes = parseInt(strTime.slice(-2), 10) / 60;
-  const hours = parseInt(strTime.substring(1), 10);
+  const minutes = parseInt(time.slice(-2), 10) / 60;
+  const hours = parseInt(time.substring(0,2), 10);
   return hours + minutes;
 }; 
 
- const calculateUnpaidSessions = () => {
-   if(unpaidSessionsData.data) {
-     const TotalAmountDue = unpaidSessionsData.data.unpaidSessions.map(time=> {
-     const { endTime, startTime, hourlyTuition } = time.course
+ const unpaidSessionsObj = () => {
```

Comment:
So this is an array of unpaid sessions (not an "Obj")? I'm a bit confused since you're returning parentObj (which is an array, since data.unpaidSessions.map() returns an array) on line 259, so what are you doing here? 

It looks like you're returning an array of unpaid sessions based on `data.unpaidSessions`. Rename your variables so that this is more clear. Also, if you're just returning `parentObj` you can just return your `data.unpaidSessions.map()`, so it would just be `const unpaidSessionsList = data.unpaidSession.map(...);`

Reply:
A review of array's .map() function would be good 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map